bin/main.rb

```diff
@@ -1,47 +1,58 @@
 #!/usr/bin/env ruby
-# rubocop:disable Style/Next
 
-puts 'Welcome to the game of Tic Tac Toe!!'
-puts 'Player 1 what is your name?'
-player1 = gets.chomp
-puts "Welcome #{player1} choose your game symbol: X or O"
-player_1_symbol = gets.chomp
-# player 1 chooses "X or O"
-puts "Ok #{player1} you will be playing with #{player_1_symbol} as your symbol"
-puts 'Player 2 what is your name?'
-player2 = gets.chomp
-puts "Welcome #{player2} choose your game symbol"
-player_2_symbol = gets.chomp
-# player 2 chooses "O or X"
-puts "Ok #{player2} you will be playing with #{player_2_symbol} as your symbol"
+require './lib/player.rb'
+require './lib/board.rb'
+require './lib/game.rb'
 
-# Display board
-puts '___1_|__2__|__3__'
-puts '___4_|__5__|__6__'
-puts '___7_|__8__|__9__'
+puts 'Welcome to Tic Tac Toe'
+def start
+  @game_symbols = %w[X O]
+  puts 'player 1 what is your name?'
+  pl_one = gets.chomp
```

Comment:
You are not validating this input. I can type the same name as the other player or leave it empty. THis shouldn't happen.